streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -433,6 +433,9 @@
     public static final String ROCKSDB_CONFIG_SETTER_CLASS_CONFIG = "rocksdb.config.setter";
     private static final String ROCKSDB_CONFIG_SETTER_CLASS_DOC = "A Rocks DB config setter class or class name that implements the <code>org.apache.kafka.streams.state.RocksDBConfigSetter</code> interface";
 
+    /** {@code rocksdb.shared.block.cache} */
+    public static final String ROCKSDB_SHARED_BLOCK_CACHE = "rocksdb.shared.block.cache";
```

Comment:
Is it really necessary to make this rocksDB config it a top-level Streams config?

Maybe we can just change the default values of rocksDB config to `shared = true` and document that users who see perf hits should consider overriding it themselves.